protocols/imap/src/main/java/org/apache/james/imap/processor/StatusProcessor.java

```diff
@@ -104,8 +104,8 @@ protected Mono<Void> processRequestReactive(StatusRequest request, ImapSession s
             .then(unsolicitedResponses(session, responder, false))
             .then(Mono.fromRunnable(() -> okComplete(request, responder)))
             .onErrorResume(MailboxException.class, e -> {
```

Comment:
Should this be explicitly `MailboxNotFoundException`?

I see lots of sub classes of `MailboxException`. For the rest, I think we still can fallback to `STATUS_FAILED`.

Reply:
Very right.

Sorry I have been too fast.